roles/installer/templates/deployments/task.yaml.j2

```diff
@@ -349,6 +378,15 @@ spec:
 {% if rsyslog_command %}
           command: {{ rsyslog_command }}
 {% endif %}
+{% if rsyslog_host_network %}
+          hostNetwork: {{ rsyslog_host_network }}
+{% endif %}
+{% if rsyslog_dns_policy %}
+         dnsPolicy: {{ rsyslog_dns_policy }}
+{% endif %}
+{% if rsyslog_dns_policy == 'None' and rsyslog_dns_config is defined }}
```

Comment:
Similarly, there is an extra closing brace at the end of this line too which causes syntax errors when applied.